turbo.json

```diff
@@ -41,6 +45,25 @@
       "dependsOn": ["^graphql:gen"],
       "inputs": [".env*"],
       "outputs": ["shared/services/api/generated/**"]
+    },
+    "transit": {
+      "dependsOn": ["^transit"]
+    },
+    "//#format": {
+      "outputs": ["node_modules/.cache/.prettiercache"]
+    },
```

Comment:
Where did this solution come from? Can you point me to the docs?

Reply:
https://prettier.io/blog/2023/11/30/cli-deep-dive#caching

Reply:
But what is this `transit` and `//#format` command? I've never seen that in Turbo.

Reply:
The task with double slashes is [a root task](https://turbo.build/repo/docs/crafting-your-repository/configuring-tasks#registering-root-tasks). 
[The `transit` task](https://turbo.build/repo/docs/crafting-your-repository/configuring-tasks#dependent-tasks-that-can-be-ran-in-parallel) is turbo magic to make tasks run in parallel (but I only saw that happen once).